tests/unit/specs/components/Step.spec.js

```diff
@@ -0,0 +1,170 @@
+import { shallowMount, createLocalVue, config } from '@vue/test-utils'
+import Step from '@/components/Step'
+import Buefy from 'buefy'
+import VueI18n from 'vue-i18n'
+import Vuex from 'vuex'
+import Vue from 'vue'
+import store from '@/store'
+
+describe('Step.vue', () => {
+    let wrapper
+    let localVue
+
+    // Always creates a shallow instance of component
+    beforeEach(() => {
+        localVue = createLocalVue()
+        localVue.use(Vuex)
+        localVue.use(Buefy)
+        Vue.use(VueI18n)
+        const i18n = new VueI18n({
+            locale: 'en',
+            fallbackLocale: 'en',
+            messages: {}
+        })
+
+        config.mocks.i18n = i18n
+
+        config.mocks.$t = key => key
```

Comment:
If we return key, it is added to html as is (eg. `stepper.SA.not-selected`), which is enough for testing.